app/elements/project-chart.html

```diff
@@ -193,21 +194,21 @@
   },
 
   _controlUpdate: function() {
-    var lowValue = this.control.getState().lowValue;
-    var highValue = this.control.getState().highValue;
-
-    var lowDate = new Date(lowValue.getFullYear(), lowValue.getMonth(), 1, 1, 1, 1);
-    var highDate = new Date(highValue.getFullYear(), highValue.getMonth() + 1, 0);
-
-    if (lowDate.getMonth() == highDate.getMonth()) {
-      var highDate = new Date(highValue.getFullYear(), highValue.getMonth() + 2, 0);
-    }
-
-    this.control.setState({
-      lowValue: lowDate,
-      highValue: highDate
-    });
-    this.control.draw();
+    // var lowValue = this.control.getState().lowValue;
```

Comment:
What is the reason to keep all function code commented? If You don't need it just remove whole function.